### ISSUE TITLE
Future MC 0.1.11+ Biome Bundle crash at the creation of a new world.

### ISSUE THREAD

**Issue by Naetheline** [thedarkcolour/Future-MC#58]
Using Biome Bundle v6.1 and FutureMC 0.1.11 makes the game crash at the creation of a new world.

The error is : The game crashed whilst exception in server tick loop
Error: java.lang.NullPointerException: Exception in server tick loop

Step to reproduce : 

- install Biome Bundle + OTG
- install Future MC
- Launch the game and create a new world.

And here are the complete log : 
```
[10:26:02] [Server thread/ERROR]: Exception stopping the server
java.lang.NullPointerException: null
	at net.minecraftforge.registries.ForgeRegistry.getOverrideOwners(ForgeRegistry.java:784) ~[ForgeRegistry.class:?]
	at net.minecraftforge.registries.ForgeRegistry.makeSnapshot(ForgeRegistry.java:771) ~[ForgeRegistry.class:?]
	at net.minecraftforge.registries.RegistryManager.lambda$takeSnapshot$0(RegistryManager.java:135) ~[RegistryManager.class:?]
	at net.minecraftforge.registries.RegistryManager$$Lambda$183/2073124154.accept(Unknown Source) ~[?:?]
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:1.8.0_51]
	at net.minecraftforge.registries.RegistryManager.takeSnapshot(RegistryManager.java:135) ~[RegistryManager.class:?]
	at net.minecraftforge.fml.common.FMLContainer.getDataForWriting(FMLContainer.java:126) ~[FMLContainer.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.handleWorldDataSave(FMLCommonHandler.java:395) ~[FMLCommonHandler.class:?]
	at net.minecraft.world.storage.SaveHandler.func_75755_a(SaveHandler.java:141) ~[bfb.class:?]
	at net.minecraft.world.chunk.storage.AnvilSaveHandler.func_75755_a(AnvilSaveHandler.java:40) ~[bey.class:?]
	at net.minecraft.world.WorldServer.func_73042_a(WorldServer.java:1051) ~[oo.class:?]
	at net.minecraft.world.WorldServer.func_73044_a(WorldServer.java:1000) ~[oo.class:?]
	at net.minecraft.server.MinecraftServer.func_71267_a(MinecraftServer.java:408) ~[MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71267_a(IntegratedServer.java:252) ~[chd.class:?]
	at net.minecraft.server.MinecraftServer.func_71260_j(MinecraftServer.java:446) ~[MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71260_j(IntegratedServer.java:372) ~[chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:579) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_51]
[10:26:02] [Client thread/INFO]: [net.minecraft.init.Bootstrap:func_179870_a:553]: ---- Minecraft Crash Report ----

WARNING: coremods are present:
  OTGCorePlugin (OTG-Core.jar)
Contact their authors BEFORE contacting forge

// This doesn't make any sense!

Time: 10/8/19 10:26 AM
Description: Exception in server tick loop

java.lang.NullPointerException: Exception in server tick loop
	at net.minecraftforge.registries.ForgeRegistry.getOverrideOwners(ForgeRegistry.java:784)
	at net.minecraftforge.registries.ForgeRegistry.makeSnapshot(ForgeRegistry.java:771)
	at net.minecraftforge.registries.RegistryManager.lambda$takeSnapshot$0(RegistryManager.java:135)
	at net.minecraftforge.registries.RegistryManager$$Lambda$183/2073124154.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at net.minecraftforge.registries.RegistryManager.takeSnapshot(RegistryManager.java:135)
	at net.minecraftforge.fml.common.FMLContainer.getDataForWriting(FMLContainer.java:126)
	at net.minecraftforge.fml.common.FMLCommonHandler.handleWorldDataSave(FMLCommonHandler.java:395)
	at net.minecraft.world.storage.SaveHandler.func_75755_a(SaveHandler.java:141)
	at net.minecraft.world.chunk.storage.AnvilSaveHandler.func_75755_a(AnvilSaveHandler.java:40)
	at net.minecraft.world.WorldServer.func_73042_a(WorldServer.java:1051)
	at net.minecraft.world.WorldServer.func_73044_a(WorldServer.java:1000)
	at net.minecraft.server.MinecraftServer.func_71267_a(MinecraftServer.java:408)
	at net.minecraft.server.integrated.IntegratedServer.func_71267_a(IntegratedServer.java:252)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:170)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1072899952 bytes (1023 MB) / 1488453632 bytes (1419 MB) up to 2863661056 bytes (2731 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx3072m -Xms256m -XX:PermSize=256m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.5.2847 8 mods loaded, 8 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State  | ID                   | Version      | Source                               | Signature                                |
	|:------ |:-------------------- |:------------ |:------------------------------------ |:---------------------------------------- |
	| LCHIJA | minecraft            | 1.12.2       | minecraft.jar                        | None                                     |
	| LCHIJA | mcp                  | 9.42         | minecraft.jar                        | None                                     |
	| LCHIJA | FML                  | 8.0.99.99    | forge-1.12.2-14.23.5.2847.jar        | None                                     |
	| LCHIJA | forge                | 14.23.5.2847 | forge-1.12.2-14.23.5.2847.jar        | None                                     |
	| LCHIJA | otgcore              | 1.12.2 - v7  | minecraft.jar                        | None                                     |
	| LCHIJA | openterraingenerator | v6           | OpenTerrainGenerator-1.12.2 - v6.jar | e9f7847a78c5342af5b0a9e04e5abc0b554d69e0 |
	| LCHIJA | biomebundle          | 5.1          | Biome_Bundle-1.12.2-v6.1.jar         | None                                     |
	| LCHIJA | minecraftfuture      | 0.1.11       | future-mc-0.1.11.jar                 | None                                     |

	Loaded coremods (and transformers): 
OTGCorePlugin (OTG-Core.jar)
  com.pg85.otg.forge.asm.OTGClassTransformer
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Player Count: 1 / 8; [EntityPlayerMP['Nimel'/13, l='Biome Bundle', x=885.50, y=47.00, z=541.50]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

```

As Biome Bundle runs fine with all my others mods, I suspect the error is in future MC.

**Comment by thedarkcolour** [comment]
Disable bees. This error has happened so many times, with a bunch of different mods. I am currently testing a solution with ASM, but it messes up when minecraft compiles. If you would like to see a preview of this, download 0.1.12pre. Keep in mind that the beehives might not generate as frequently and will spawn on weird spots of the trees. Campfire functionality is also in 0.1.12pre, and so is JEI support. I think that it's not a bad tradeoff to update now, but if you are uncomfortable with the weird Beehive generation then just turn off bees. 0.1.12pre DOES NOT crash with bees enabled, but 0.1.11 does.

**Comment by thedarkcolour** [comment]
I will post 0.1.12 full update when I reach a solution.

**Comment by thedarkcolour** [comment]
Fixed in 0.1.12
